.sync.yml

```diff
@@ -7,60 +7,23 @@
   unmanaged: true
 
 .travis.yml:
-  docker_sets:
-    - set: docker/centos-7
-    - set: docker/ubuntu-14.04
-  docker_defaults:
-    bundler_args: ""
-    script: bundle exec rake beaker
-  secure: ""
-  global_env:
-    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
-    - GEM_BOLT=true
-  branches:
-    - release
-  remove_includes:
-    - env: PUPPET_GEM_VERSION="~> 5.0" CHECK=parallel_spec
-      rvm: 2.4.5
-      stage: spec
+  unmanaged: true
 
-.yardopts:
-  optional:
-    - --output-dir docs/
+appveyor.yml:
+  unmanaged: true
 
 Gemfile:
   optional:
     ':development':
-      - gem: 'github_changelog_generator'
-        git: 'https://github.com/skywinder/github-changelog-generator'
-        ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018'
-        condition: "Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')"
-  required:
-    ':system_tests':
-      - gem: 'puppet-module-posix-system-r#{minor_version}'
-        platforms: ruby
-      - gem: 'puppet-module-win-system-r#{minor_version}'
-        platforms:
-          - mswin
-          - mingw
-          - x64_mingw
-      - gem: bolt
-        version: '~> 1.15'
-        condition: ENV['GEM_BOLT']
+        - gem: 'github_changelog_generator'
+          git: 'https://github.com/skywinder/github-changelog-generator'
+          ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018'
+          condition: "Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')"
+      
 
 Rakefile:
   requires:
-    - puppet-lint/tasks/puppet-lint
-  extras:
-  extras: |
```

Comment:
Currently without this task the acceptance test CI does not know how to execute beaker tests. Are we moving away from jenkins entirely? https://jenkins-master-prod-1.delivery.puppetlabs.net/job/forge-module_puppetlabs-package_intn-sys_smoke-adhoc/PLATFORM=redhat8-64default.a,WORKER_LABEL=beaker/42/console ?

I seem to recall that the reason bolt was conditionally required with an environment variable was because in the push to forge pipelines, installing it would conflict with another gem (maybe puppet), Assuming bolt is a dependency of litmus will we expect to have the same issues?

Reply:
yeah. We are moving away from Jenkins on executing tests. But we use Jenkins to push modules to forge. Was talking with team also looks like that has been fixed and we have been able to release litmus converted modules.